release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -17,7 +17,7 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 {product-title} (link:https://access.redhat.com/errata/RHSA-2022:TODO[RHSA-2022:TODO]) is now available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md[Kubernetes 1.24] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
 //Red Hat did not publicly release {product-title} 4.10.0 as the GA version and, instead, is releasing {product-title} 4.10.3 as the GA version.
 
-{product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title} users can you to deploy OpenShift clusters to either on-premise or cloud environments.
+{product-title} {product-version} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for {product-title} users can you to deploy OpenShift clusters to either on-premises or cloud environments.
```

Comment:
Word choice/sentence structure. 

There's some minor errors in the last sentence. Rephrase to something like:

`With the {cluster-manager-first} application for {product-title}, you can deploy OpenShift clusters to either on-premises or cloud environments.`

Reply:
Excellent! Thanks!